doc/ch-image.rst

```diff
@@ -313,11 +319,15 @@ repositories within the image should work. **Important exception**: No files
 named :code:`.git*` or other Git metadata are permitted in the image’s root
 directory.
 
-`Extended attributes <https://man7.org/linux/man-pages/man7/xattr.7.html>`_ (xattrs)
-belonging to unprivileged namespaces (e.g. :code:`user`) are also saved and
-restored by the cache by default. Notably, extended attributes in privileged
-namespaces (e.g. :code:`trusted`) cannot be read by :code:`ch-image` and will be
-lost without warning.
+`Extended attributes <https://man7.org/linux/man-pages/man7/xattr.7.html>`_
+(xattrs) are ignored by the build cache by default. Cache support for xattrs
+belonging to unprivileged namespaces (e.g. :code:`user`) can be enabled by
```

Comment:
"in unprivileged xattr namespaces" to clarify we're not talking about container namespaces?